decidim-core/spec/forms/account_form_spec.rb

```diff
@@ -154,5 +156,23 @@ module Decidim
         end
       end
     end
+
+    describe "time_zone" do
+      context "when an empty time_zone" do
+        let(:time_zone) { "" }
+
+        it "is invalid" do
+          expect(subject).not_to be_valid
```

Comment:
```suggestion
          expect(subject).to be_valid
```
Should be valid in this case.

We should add a case when timezone is invalid (not a valid time zone, like in the suggestion for the `update_account_spec.rb`